### DISCUSSION TITLE
GODOTEX Help please

### DISCUSSION THREAD

**Discussion by ProfessorBE** [lennart-finke/GodoTeX#20] [General]
So I am attempting to get GODOTEX to work and having a hard time.  As it stands I can do the install and get it running within the IDE.

I have a database and as you swipe around on the screen, it accesses different database items written in latex and puts them on the screen.

When I export to android it has issues.  Somehow, it will render the first item which is set to happen  with _ready

However when doing that initial rendering, the error code it throws is

The type initializer for SkiaSharp.SKImageInfo' threw an exception. --> System.DllNotFoundException: libSkiaSharp

Then as I swipe around each swipe adjusts the data base item, when then calls the renderer and nothing changes I just get this error message.

Again, when in the Editor, it works perfectly.  Exporting it doesn't.  But I don't get how on ready it has no issue rendering, but when calling the renderer inside a subruoutine it freaks out.

**Comment by lennart-finke** [comment]
Thanks for using GodoTeX! As stated in the project readme, sadly the only expected platforms to currently work are Windows, Linux and macOS. For a discussion thread by someone who started looking exporting to Android, see #5.
